src/compat/glibcxx_sanity.cpp

```diff
@@ -47,7 +47,8 @@ bool sanity_test_range_fmt()
 {
     std::string test;
     try {
-        test.at(1);
+        char ignore = test.at(1);
```

Comment:
looks like std::ignore could be used here

Reply:
It comes with <tuple> headers. Do not want to bring all tuple related stuff for such a small fix.